iotile_analytics_interactive/iotile_analytics/interactive/scripts/analytics_host.py

```diff
@@ -113,6 +114,10 @@ def build_args():
     parser.add_argument('-c', '--no-confirm', action="store_true", help="Do not confirm the analysis that you are about to perforn and prompt for parameters")
     parser.add_argument('-l', '--list', action="store_true", help="List all known analysis types without running one")
     parser.add_argument('-b', '--bundle', action="store_true", help="Bundle the rendered output into a zip file")
+    parser.add_argument('-w', '--web-push', action="store_true", help="Push the resulting report to iotile.cloud.")
+    parser.add_argument('--web-push-label', type=str, default=None, help="Set the label used when pushing a report to iotile.cloud")
+    parser.add_argument('--web-push-org', type=str, default=None, help="Override the org given in the analysisgroup and force it to be this")
```

Comment:
The org should not be needed, as it can be inferred by the device/block.
I assume we don't want to allow uploading to a different org

Reply:
Good point.  Let me refactor this.